index.html

```diff
@@ -30,5 +30,199 @@ <h1 class="header header-1" id="job">I'm a software developer</h1>
+                    </div>
+                    <div class="sample-img-txt-head-container" id="sample-img-4-text-head-container">
+                        <h3 class="sample-img-head-txt" id="sample-img4-head-txt">Multi-Post Stories Gain+Glory</h3>
+                    </div>
+                    <div class="code-lang-container" id="code-lang-container-4">
+                        <div class="ruby-on-rails">Ruby on rails</div>
+                        <div class="css">css</div>
+                        <div class="js">Javascript</div>
+                        <div class="html">html</div>
+                    </div>
+                    <div class="btn-container">
+                        <button type="button" class="btn">See Project</button>
+                    </div>
+                </div>
+                <div class="container sample-containers" id="sample-5-container">
+                    <div class="sample-img-container" id="sample-5-img-container">
+                        <img src="#" alt="" class="sample-img" id="sample-img-5">
+                    </div>
+                    <div class="sample-img-txt-head-container" id="sample-img-5-text-head-container">
+                        <h3 class="sample-img-head-txt" id="sample-img-5-head-txt">Multi-Post Stories Gain+Glory</h3>
+                    </div>
+                    <ul class="container code-lang-container" id="code-lang-container-5">
+                        <li class="ruby-on-rails">Ruby on rails</li>
+                        <li class="css">css</li>
+                        <li class="js">Javascript</li>
+                        <li class="html">html</li>
+                    </ul>
+                    <div class="btn-container">
+                        <button type="button" class="btn">See Project</button>
+                    </div>
+                </div>
+                <div class="sample-containers" id="sample-6-container">
+                    <div class="sample-img-container" id="sample-6-img-container">
+                        <img src="#" alt="" class="sample-img" id="sample-img-6">
+                    </div>
+                    <div class="sample-img-txt-head-container" id="sample-img-6-text-head-container">
+                        <h3 class="sample-img-head-txt" id="sample-img-6-head-txt">Multi-Post Stories Gain+Glory</h3>
+                    </div>
+                    <div class="code-lang-container" id="code-lang-container-6">
+                        <div class="ruby-on-rails">Ruby on rails</div>
+                        <div class="css">css</div>
+                        <div class="js">Javascript</div>
+                        <div class="html">html</div>
+                    </div>
+                    <div class="btn-container">
+                        <button type="button" class="btn">See Project</button>
+                    </div>
+                </div>
+            </article>
+        </main>
+        <aside class="container" id="aside-container">
```

Comment:
- [ ] Once again to keep our code well organized kindly ensure to enclose the 'about me' section with a `section` tag.

Reply:
- [ ] As pointed out by the previous TSE you have to wrap The "about me" section inside a `section` tag but you haven't still addressed that case. Please try and fix that.

Reply:
I have made the changes.

Reply:
Well resolved